cmd/tendermint/commands/key_migrate.go

```diff
@@ -0,0 +1,64 @@
+package commands
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/spf13/cobra"
+	cfg "github.com/tendermint/tendermint/config"
+	"github.com/tendermint/tendermint/scripts/keymigrate"
+)
+
+func MakeKeyMigrateCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "key-migrate",
+		Short: "Run Database key migration",
```

Comment:
I'm wondering if we need to also add a `Long:` description (basically what you said in the upgrades document for those that want extra context

Reply:
I hope that people are going to be looking at the upgrade document regardless of what this says, and I think that it would be easy to loose a few of the details.

Reply:
this is very vague. Someone looking at this command could be confused as to why they need to do this. There is no mention of looking at the upgrading doc as well. Unless the person is psychic how will they know to look at that doc and we cant expect every to do this.